node/src/p2p/swarm.rs

```diff
@@ -73,6 +74,14 @@ mod imp {
                 webtransport_websys::Transport::new(config)
             })
             .expect("webtransport_websys::Transport is infallible")
+            .with_other_transport(|local_keypair| {
+                let noise_config = noise::Config::new(local_keypair)?;
+                Ok(websocket_websys::Transport::default()
+                    .upgrade(Version::V1)
+                    .authenticate(noise_config)
+                    .multiplex(yamux::Config::default()))
+            })
+            .map_err(|_| P2pError::InitNoise("websocket-websys".to_string()))?
```

Comment:
```suggestion
            .map_err(|e| P2pError::InitNoise(format!("websocket-websys: {e}")))?
```

Reply:
I'm not sure it's worth it as we call it just once, and cache won't cross workers boundary